tools/release/sync-gutenberg-packages.js

```diff
@@ -107,6 +118,13 @@ function getMismatchedNonWordPressDependencies() {
 	const currentPackages = getWordPressPackages( currentPackageJSON );
 
 	const packageLock = readJSONFile( "package-lock.json" );
+	console.log("JT   ->");
```

Comment:
Is this chunk related to the secondary issue you noted? I think we should split that out into a different bug report on Trac to separate it out.